user/TASK/chassis_task/chassis_task.h

```diff
@@ -17,7 +17,7 @@
 #include "pid.h"
 
 /******************************* Task Delays *********************************/
-#define CHASSIS_TASK_DELAY 1
+#define CHASSIS_TASK_DELAY 5
 #define CHASSIS_INIT_DELAY 20
```

Comment:
Maybe we should mention the units, ms

Reply:
Are we sure.... Isn't this reliant on the RTOS click rate?

Reply:
Turns out its the number of ticks - my bad